ytmusicapi/parsers/_utils.py

```diff
@@ -67,7 +67,11 @@ def parse_duration(duration):
     # duration may be falsy or a single space: ' '
     if not duration or not duration.strip():
         return duration
-    mapped_increments = zip([1, 60, 3600], reversed(duration.split(":")))
+    duration_split = duration.strip().split(":")
+    for d in duration_split:
+        if not d.isdigit(): # For e.g: "2,343"
+            return
```

Comment:
Did you mean to return `None` ?

Reply:
Both `return` and `return None` are the same in python

Reply:
True, but being explicit is preferred here

https://stackoverflow.com/a/15300671